### PR TITLE
updates lobbyscreen art to RGA

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR 
Removes the poor copyright.txt and changes it to robust generic attribution.

### CODE REVIEW

File: Resources/Textures/LobbyScreens/attributions.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- files: ["supermatter.png", "robotics.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Veritius#2351 on discord"
```
Comment (Veritius) [inline Resources/Textures/LobbyScreens/attributions.yml:3]: Can you do `Veritius#2351 (257233913951289344) on discord / @Veritius on GitHub`

**Review by Veritius — APPROVED** [review]

File: Resources/Textures/LobbyScreens/attributions.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- files: ["supermatter.png", "robotics.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Veritius#2351 (257233913951289344) on discord / @Veritius on GitHub"
+  source: ""
+  
+- files: ["warden.png", "pharmacy.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Solbusaur#4343 on discord"
```
Comment (ike709) [inline Resources/Textures/LobbyScreens/attributions.yml:8]: Should probably include their actual GUID?

Comment (Peptide90) [reply Resources/Textures/LobbyScreens/attributions.yml:8]: I pinged them, no response. Where would I get their GUID?

Comment (ike709) [reply Resources/Textures/LobbyScreens/attributions.yml:8]: Pinged you instructions on discord

**Review by ike709 — APPROVED** [review]

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**